.github/scripts/util.py

```diff
@@ -141,13 +142,9 @@ def sortListings(listings):
 
     listings.sort(
         key=lambda x: (
-            datetime(
-                datetime.fromtimestamp(x["date_posted"]).year,
-                datetime.fromtimestamp(x["date_posted"]).month,
-                datetime.fromtimestamp(x["date_posted"]).day
-            ),
+            x["active"],  # Active listings first
+            datetime.fromtimestamp(x["date_posted"]),
```

Comment:
This doesn't strip out hours, minutes, seconds, and microseconds from the timestamp, so it won't group postings from the same company on the same day together.
```suggestion
            datetime(
                datetime.fromtimestamp(x["date_posted"]).year,
                datetime.fromtimestamp(x["date_posted"]).month,
                datetime.fromtimestamp(x["date_posted"]).day
            ),
```